stages/00-commands.sh

```diff
@@ -31,6 +31,15 @@ INPLACE() {
   :
 }
 
+
+# INCLUDE sources the provided Pifile
+#
+# Usage: INCLUDE path/to/pifile[.Pifile]
+INCLUDE() {
+  filename="${$1%.*}"
```

Comment:
There is a typo in the substitution resulting in an error.

```suggestion
  filename="${1%.*}"
```